### DISCUSSION TITLE
Custom font as a configurable option

### DISCUSSION THREAD

**Discussion by ganeshkrishnasharma** [chakra-ui/chakra-ui#9071] [General]
I am looking to build a SDK, where i want my customer to pass the font url as a config to my init method. With v3 whats the best way to use this input and set the custom font?

**Comment by segunadebayo** [comment]
Accept the URL, inject `@font-face` or an `@import` (Emotion `Global` or `globalCss`), then point `theme.tokens.fonts.body` / `heading` at that family.

The token is just the CSS `font-family` name. Loading the file is separate.
